tests/test_cli.py

```diff
@@ -175,3 +174,22 @@ def test_unused_found_because_excluded_but_not_installed(  # noqa: PLR0913
 
     assert captured.out.splitlines() == expected_unused_packages
     assert exit_code == expected_exit_code
+
+
+def test_load_defaults_no_file(tmp_path):
+    missing_pyproject = tmp_path / "pyproject.toml"
+    config = cli.load_defaults(missing_pyproject)
+    assert config == cli.Config()  # Should return a default config
```

Comment:
It would be great if you could complement this test with tests where the pyproject.toml is not empty, and:
- contains a full config and asserts on all the values.
- contains a partial config and asserts on all the values (of which some will use non-default and default values).

Reply:
I see you added this in the PR description:

```toml
[tool.creosote]
venvs=[".venv"]
paths=["src"]
deps-file="pyproject.toml"
sections=["tool.poetry.dependencies"]
exclude-deps =[
  "importlib_resources",
  "pydantic",
]
```

It would be very nice if the assertions would show both the keys and the values, so we don't make any mistakes here. I'm thinking about the fact that there's a dash in `deps-file` for example, which I think is likely going to become an underscore.

Reply:
addressed in 23a9f682c6fcd9c8ffd89e